### PR TITLE
Feat: Added Playlist Managment

### PR DESCRIPTION
**Pull request by Asuraking1n**
1- Create Playlist
2-Delete Playlist
3-Add Videos to playlist
4-Visit Playlist

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { BrowserRouter as Router } from "react-router-dom";
 import { VideoProvider } from "./context/vedio-data-context";
 import { HistoryProvider } from "./context/history-context";
 import { LikeProvider } from "./context/like-context";
+import {PlayListProvider} from './context/playlist-context'
```
Comment (vikrantkode) [inline src/index.js:10]: You can import all context from one file

**Review by vikrantkode — APPROVED** [review]
Just a small change suggested. You can look into it.
- project Structure and code is clean
-

File: src/index.js (hunk 1 above)
Comment (Asuraking1n) [reply src/index.js:10]: Sure, i will update

File: src/pages/playList/SinglePlayList.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React,{useEffect,useState} from 'react'
+import './playlist.css'
+import { Link,useParams } from "react-router-dom";
+import axios from 'axios';
+import VedioCard from '../../components/cards/vedioCards/VedioCard'
```
Comment (krgarima) [inline src/pages/playList/SinglePlayList.js]: spelling mistake 'video'

**Review by krgarima — APPROVED** [review]
Looks good to me. Just a suggestion, you can highlight the page category which is open.

**Pull request merged**